src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Models/CmdletParamEnums.cs

```diff
@@ -14,6 +14,13 @@
 
 namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.Models
 {
+    public enum VaultParams
+    {
+        VaultName,
+        ResourceGroupName,
+        VaultLocation,
```

Comment:
I don't see this used anywhere - is it needed?

Reply:
This is being used in the Restore cmdlet. You can find this change at https://github.com/Azure/azure-powershell/pull/6327/files#diff-b7f777ff31af1c21fbf6378a2f1e1647

Reply:
This is being used in the Restore cmdlet. https://github.com/Azure/azure-powershell/pull/6327/files#diff-b7f777ff31af1c21fbf6378a2f1e1647